[BUG] Test_genOption failed in CI

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

```
--- FAIL: Test_genOption (0.00s)
    --- FAIL: Test_genOption/test (0.00s)
        transform_fuse_test.go:763: genOption() = [c a=b], want [a=b c]
        transform_fuse_test.go:763: genOption() = [c a=b], want [a=b c]
1.651850044261682e+09	DEBUG	test	Nothing to do for syncing
1.6518500442662694e+09	INFO	test1	workers are not ready	{"phase": "NotReady"}
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/charts/fluid/fluid/Chart.yaml
+++ b/charts/fluid/fluid/Chart.yaml
@@ -18,7 +18,7 @@ version: 0.8.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 0.8.0-6a5acd3
+appVersion: 0.8.0-d66e8f9
 home: https://github.com/fluid-cloudnative/fluid
 keywords:
   - category:data

--- a/charts/fluid/fluid/values.yaml
+++ b/charts/fluid/fluid/values.yaml
@@ -6,7 +6,7 @@ workdir: /tmp
 
 dataset:
   controller:
-    image: fluidcloudnative/dataset-controller:v0.8.0-6a5acd3
+    image: fluidcloudnative/dataset-controller:v0.8.0-d66e8f9
 
 csi:
   featureGates: "FuseRecovery=false"
@@ -15,7 +15,7 @@ csi:
   registrar:
     image: registry.aliyuncs.com/acs/csi-node-driver-registrar:v1.2.0
   plugins:
-    image: fluidcloudnative/fluid-csi:v0.8.0-6a5acd3
+    image: fluidcloudnative/fluid-csi:v0.8.0-d66e8f9
   kubelet:
     rootDir: /var/lib/kubelet
 
@@ -28,9 +28,9 @@ runtime:
     portRange: 20000-26000
     enabled: true
     init:
-      image: fluidcloudnative/init-users:v0.8.0-6a5acd3
+      image: fluidcloudnative/init-users:v0.8.0-d66e8f9
     controller:
-      image: fluidcloudnative/alluxioruntime-controller:v0.8.0-6a5acd3
+      image: fluidcloudnative/alluxioruntime-controller:v0.8.0-d66e8f9
     runtime:
       image: registry.aliyuncs.com/alluxio/alluxio:release-2.7.3-SNAPSHOT-a7154f1
     fuse:
@@ -44,19 +44,19 @@ runtime:
     fuse:
       image: registry.cn-shanghai.aliyuncs.com/jindofs/jindo-fuse:3.8.0
     controller:
-      image: fluidcloudnative/jindoruntime-controller:v0.8.0-6a5acd3
+      image: fluidcloudnative/jindoruntime-controller:v0.8.0-d66e8f9
     init:
       portCheck:
         enabled: false
-      image: fluidcloudnative/init-users:v0.8.0-6a5acd3
+      image: fluidcloudnative/init-users:v0.8.0-d66e8f9
   goosefs:
     runtimeWorkers: 3
     portRange: 26000-32000
     enabled: false
     init:
-      image: fluidcloudnative/init-users:v0.8.0-6a5acd3
+      image: fluidcloudnative/init-users:v0.8.0-d66e8f9
     controller:
-      image: fluidcloudnative/goosefsruntime-controller:v0.8.0-6a5acd3
+      image: fluidcloudnative/goosefsruntime-controller:v0.8.0-d66e8f9
     runtime:
       image: ccr.ccs.tencentyun.com/qcloud/goosefs:v1.2.0
     fuse:
@@ -64,16 +64,16 @@ runtime:
   juicefs:
     enabled: false
     controller:
-      image: fluidcloudnative/juicefsruntime-controller:v0.8.0-6a5acd3
+      image: fluidcloudnative/juicefsruntime-controller:v0.8.0-d66e8f9
     fuse:
       image: registry.cn-hangzhou.aliyuncs.com/juicefs/juicefs-fuse:v1.0.0-beta2
 
 webhook:
   enabled: true
-  image: fluidcloudnative/fluid-webhook:v0.8.0-6a5acd3
+  image: fluidcloudnative/fluid-webhook:v0.8.0-d66e8f9
   replicas: 1
 
 fluidapp:
   enabled: true
   controller:
-    image: fluidcloudnative/application-controller:v0.8.0-6a5acd3
+    image: fluidcloudnative/application-controller:v0.8.0-d66e8f9

--- a/pkg/ddc/juicefs/transform_fuse_test.go
+++ b/pkg/ddc/juicefs/transform_fuse_test.go
@@ -18,11 +18,10 @@ package juicefs
 
 import (
 	"encoding/base64"
-	"fmt"
+	"testing"
+
 	"github.com/fluid-cloudnative/fluid/pkg/common"
 	"github.com/go-logr/logr"
-	"sort"
-	"testing"
 
 	"k8s.io/apimachinery/pkg/api/resource"
 
@@ -752,17 +751,30 @@ func Test_genOption(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got := genOption(tt.args.optionMap)
-			var keys []int
-			for k := range got {
-				keys = append(keys, k)
-			}
-			sort.Ints(keys)
-			fmt.Println(keys)
-			for _, k := range keys {
-				if got[k] != tt.want[k] {
-					t.Errorf("genOption() = %v, want %v", got, tt.want)
-				}
+			if !isSliceEqual(got, tt.want) {
+				t.Errorf("genOption() = %v, want %v", got, tt.want)
 			}
 		})
 	}
+}
+
+func isSliceEqual(got, want []string) bool {
+	if len(got) != len(want) {
+		return false
+	}
+
+	diff := make(map[string]int, len(got))
+	for _, v := range got {
+		diff[v]++
+	}
+	for _, v := range want {
+		if _, ok := diff[v]; !ok {
+			return false
+		}
+		diff[v] -= 1
+		if diff[v] == 0 {
+			delete(diff, v)
+		}
+	}
+	return len(diff) == 0
 }

